internal/pod_client.go

```diff
@@ -166,14 +165,6 @@ func (client *realFdbPodClient) makeRequest(method string, path string) (string,
 	retryClient.RetryWaitMax = 1 * time.Second
 	// Prevent logging
 	retryClient.Logger = nil
-	retryClient.CheckRetry = func(c context.Context, resp *http.Response, err error) (bool, error) {
```

Comment:
I would prefer us to use `retryablehttp.ErrorPropagatedRetryPolicy()` so that we can log what happened after we exhaust our retries. Check the use here: https://github.com/hashicorp/go-retryablehttp/blob/master/client_test.go#L252

Reply:
That sounds reasonable and the `retryablehttp.ErrorPropagatedRetryPolicy` is a drop-in replacement.